processor/src/trace/virtual_bus/circuit/prover.rs

```diff
@@ -198,56 +201,43 @@ impl<E: FieldElement> Layer<E> {
 /// - c -> `right_numerators`
 /// - d -> `right_denominators`
 pub struct LayerPolys<E: FieldElement> {
-    pub left_numerators: MultiLinearPoly<E>,
-    pub right_numerators: MultiLinearPoly<E>,
-    pub left_denominators: MultiLinearPoly<E>,
-    pub right_denominators: MultiLinearPoly<E>,
+    pub numerators: MultiLinearPoly<E>,
+    pub denominators: MultiLinearPoly<E>,
 }
 
-impl<E: FieldElement> From<Layer<E>> for LayerPolys<E> {
-    fn from(layer: Layer<E>) -> Self {
+impl<E: FieldElement> From<CircuitLayer<E>> for LayerPolys<E> {
+    fn from(layer: CircuitLayer<E>) -> Self {
         layer.nodes.into()
     }
 }
 
-impl<E> From<Vec<ProjectiveCoordinates<E>>> for LayerPolys<E>
+impl<E> From<Vec<Node<E>>> for LayerPolys<E>
```

Comment:
Note: I will make this a `LayerPolys` method or something like that in my PR after we merge this